networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,15 +13,56 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-3"]
+== Network Observability Operator 1.3.0
+The following advisory is available for the Network Observability Operator 1.2.0:
+
+* link:https://access.redhat.com/errata/RHSA-2023:110622[RHSA-2023:110622 Network Observability Operator 1.3.0]
+
+[id="network-observability-operator-1.3.0-features-enhancements"]
+=== New features and enhancements
+
+[id="multi-tenancy-1.3"]
+==== Multi-tenancy in Network Observability 
+* System administrators can allow and restrict individual user access, or group access, to the flows stored in Loki. Fore more information, xref:../../networking/network_observability/installing-operators.adoc#network-observability-multi-tenancy_network_observability[Multi-tenancy in Network Observability].
+
+[id="flow-based-dashboard-1.3"]
+==== Flow-based metrics dashboard 
+* This release adds new a dashboard, which provides an overview of the network flows in your {product-title} cluster. For more information, see xref:../../networking/network_observability/network-observability-overview.adoc#network-observability-dashboards[Network Observability metrics].
+
+[id="must-gather-1.3"]
+==== Troubleshooting with the must-gather tool
+* You can now use must-gather to troubleshoot the Network Observability Operator. For more information, see xref:../../networking/network_observability/troubleshooting-network-observability.adoc#network-observability-must-gather[Network Observability must-gather].
+
+[id="multi-arch-1.3"]
+==== Multiple architectures now supported 
+* The Network Observability Operator now supports multiple architectures: `amd64`, `ppc64le`, or `arm64`. 
```

Comment:
Suggest:
Network Observability Operator can now run on an `amd64`, `ppc64le`, or `arm64` architecture.  Previously, it only ran on `x86_64`.